app/api/debug.py

```
@@ -2,17 +2,50 @@
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail="Not found",
+        )
+
 
 @router.post("/debug/log")
-async def log_message(request: Request) -> dict[str, str]:
+@limiter.limit("10/minute")
+async def log_message(
+    request: Request,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)